### PR TITLE
Sentry logs

### PR DESCRIPTION
**Pull request by avichalp**
This PR configure the logs using `senic-hub.ini` file.

### CODE REVIEW

File: senic_hub/backend/device_discovery.py [hunk 1]
```diff
@@ -22,6 +24,12 @@
 from .network_discovery import NetworkDiscovery
 
 
+if os.path.isfile('/etc/senic_hub.ini'):  # pragma: no cover
```
Comment (TheMeaningfulEngineer) [inline senic_hub/backend/device_discovery.py]: What does existence of `/etc/senic_hub.ini` means behavior wise?

Comment (avichalp) [reply senic_hub/backend/device_discovery.py]: If it is not present the loggers will have default configuration if it is there loggers will be configured using the settings it contains for the loggers.

Comment (TheMeaningfulEngineer) [reply senic_hub/backend/device_discovery.py]: > if it is there loggers will be configured using the settings it contains for the loggers.
Why do we want that?

Comment (avichalp) [reply senic_hub/backend/device_discovery.py]: Actually, we are sure that on the hardware the file will be present.
But for someone who is running tests on local machine or when tests are running on Travis the file is not present in those environments.

**Pull request merged**